File: src/linkerz_crawler_core/src/test/java/org/linker/crawler/core/TestContext.java
package org.linker.crawler.core;

import org.junit.Assert;
import org.junit.Test;
import org.junit.runner.RunWith;
import org.springframework.beans.BeansException;
import org.springframework.context.ApplicationContext;
import org.springframework.context.ApplicationContextAware;
import org.springframework.test.context.ContextConfiguration;
import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;

import static org.junit.Assert.assertEquals;

/**
 * The Class TestContext.
 *
 * @author Nguyen Duc Dung
 * @since 7/2/12, 12:26 AM
 */
@RunWith(SpringJUnit4ClassRunner.class)
@ContextConfiguration(locations = "/context.xml")
public class TestContext implements ApplicationContextAware {

    private ApplicationContext context;

    @Override
    public void setApplicationContext(ApplicationContext applicationContext) throws BeansException {
        this.context = applicationContext;
    }

    @Test
    public void test() {
        assertEquals(context != null, true);
    }
}

File: src/linkerz_crawler_core/src/test/scala/org/linkerz/crawler/core/TestContext.scala
/*
 * Copyright (C) 2012 - 2013 LinkerZ (Searching and Sharing)
 */

package org.linkerz.crawler.core

import org.linkerz.test.spring.SpringContext
import org.junit.Test

/**
 * The Class TestContext.
 *
 * @author Nguyen Duc Dung
 * @since 7/10/12, 1:04 AM
 *
 */
class TestContext extends SpringContext {

  @Test
  def testContext() {
    assert(context != null)
  }

}
